react/CozyDialogs/specific/AuthentificationDialog.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import PropTypes from 'prop-types'
+
+import PermissionDialog from '../PermissionDialog'
+import Buttons from '../../Buttons'
+import { useI18n } from '../../I18n'
+import Typography from '../../Typography'
+import { useClient } from 'cozy-client'
+import CozyAuthentificationIcon from '../../Icons/CozyAuthentification'
+
+import PasswordField from './PasswordField'
+import withSpecificDialogLocales from './locales/withSpecificDialogLocales'
+
+/**
+ * Dialog used to authenticate a user in the cozy system.
+ * The authentication logic is implemented in the applications.
+ */
+const AuthentificationDialog = ({
+  closable,
+  onDismiss,
+  onAuthentification,
```

Comment:
`onClose` and `onSubmit` serait plus générique peut-être 🤔

Reply:
Oui je m'accorde avec toi :) j'ai fait les modifs dans ce sens